### PR TITLE
Fix: incorrect json format output with `stat` command

### LINKED ISSUE #2196: `nerdctl stats --format={{json .}}` outputs invalid `CPUPerc` / `MemPerc`
### Description

When running `nerdctl stats --format='{{ json . }}'`, the `CPUPerc` and `MemPerc` fields are invalid, causing JSON parsers to fail.

(This command appears to be used in some Docker Desktop extensions.)

### Steps to reproduce the issue

1. `nerdctl run -d --restart=always nginx/stable` (or any other method of creating at least one container of your choice).
2. Run `nerdctl stats --all --no-stream --format '{{json .}}'`

### Describe the results you received and expected

```
$ nerdctl stats --all --no-stream ; nerdctl stats --all --no-stream --format '{{json .}}'
CONTAINER ID   NAME          CPU %     MEM USAGE / LIMIT   MEM %     NET I/O         BLOCK I/O         PIDS
09fa776b3a1e   nginx-09fa7   0.00%     6.363MiB / 8EiB     0.00%     2.08kB / 822B   10.5MB / 8.19kB   3
{"Name":"nginx-09fa7","ID":"09fa776b3a1e","CPUPerc":"0.00%!"(MISSING),"MemUsage":"6.363MiB / 8EiB","MemPerc":"0.00%!"(MISSING),"NetIO":"2.08kB / 822B","BlockIO":"10.5MB / 8.19kB","PIDs":"3"}
```
Expected:
```json
{"Name":"nginx-09fa7","ID":"09fa776b3a1e","CPUPerc":"0.00%","MemUsage":"6.363MiB / 8EiB","MemPerc":"0.00%","NetIO":"2.08kB / 822B","BlockIO":"10.5MB / 8.19kB","PIDs":"3"}
```

The difference is around the `(MISSING)` bits; looks like it might be `fmt.Printf` errors?

### What version of nerdctl are you using?

```
Client:
 Version:	v1.3.1
 OS/Arch:	linux/arm64
 Git commit:	b224b280ff3086516763c7335fc0e0997aca617a
 buildctl:
  Version:	v0.11.5
  GitCommit:	252ae63bcf2a9b62777add4838df5a257b86e991

Server:
 containerd:
  Version:	v1.6.8
  GitCommit:	9cd3357b7fd7218e4aec3eae239db1f68a5a6ec6
 runc:
  Version:	1.1.4
  GitCommit:	5fd4c4d144137e991c4acebb2146ab1483a97925
```

That's the `v1.3.1` tag.

### Are you using a variant of nerdctl? (e.g., Rancher Desktop)

Rancher Desktop for macOS

### Host information

```
Client:
 Namespace:	default
 Debug Mode:	false

Server:
 Server Version: v1.6.8
 Storage Driver: overlayfs
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Log: fluentd journald json-file syslog
  Storage: native overlayfs
 Security Options:
  seccomp
   Profile: default
 Kernel Version: 5.15.107-0-virt
 Operating System: Alpine Linux v3.16
 OSType: linux
 Architecture: aarch64
 CPUs: 2
 Total Memory: 1.93GiB
 Name: lima-rancher-desktop
 ID: 167b5ee9-8314-4cce-b381-75ca5cd78c8a

WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
```

### PR DESCRIPTION
**Pull request by yuchanns**
Fix #2196

### CODE REVIEW

File: cmd/nerdctl/container_stats.go [hunk 1]
```diff
@@ -329,7 +329,7 @@ func statsAction(cmd *cobra.Command, args []string) error {
 					if err := tmpl.Execute(&b, rc); err != nil {
 						break
 					}
-					if _, err = fmt.Fprintf(cmd.OutOrStdout(), b.String()+"\n"); err != nil {
+					if _, err = fmt.Fprint(cmd.OutOrStdout(), b.String()+"\n"); err != nil {
```
Comment (djdongjin) [inline cmd/nerdctl/container_stats.go:332]: `fmt.Fprintln(cmd.OutOrStdout(), b.String())`

Comment (yuchanns) [reply cmd/nerdctl/container_stats.go:332]: Make sense.

Comment (AkihiroSuda) [reply cmd/nerdctl/container_stats.go:332]: Can we merge this PR, or do you plan to update this PR?

Comment (yuchanns) [reply cmd/nerdctl/container_stats.go:332]: Plan to update. Traveling.

**Comment by djdongjin** [comment]
@yuchanns LGTM. please sign the commit, thanks

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**